doc/api.apib

```diff
@@ -46,21 +46,21 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
             }
         }
 
-## Workflow Instances [/v1/workflows/{component}/{endpoint}/instances]
+## Workflow Instances [/v1/workflows/{component}/{workflow}/instances]
 
 + Parameters
     + component: `styx-canary` (string) - Workflow Component
-    + endpoint: `LuigiCanary` (string) - Workflow Endpoint
+    + workflow: `LuigiCanary` (string) - Workflow ID
 
 ### Get Workflow Instances [GET]
 
 + Response 200 (application/json)
 
         [{
             "workflow_instance": {
-              "workflow_id": {
+              "workflow": {
                 "component_id": "styx-canary",
-                "endpoint_id": "LuigiCanary"
+                "workflow": "LuigiCanary"
```

Comment:
Looks weird to me that the "workflow" object contains a "workflow" field. Should one of these be `workflow_id`?